pkgs/by-name/hs/hsd/package.nix

```diff
@@ -40,6 +40,5 @@ buildNpmPackage rec {
     description = "Implementation of the Handshake protocol";
     homepage = "https://github.com/handshake-org/hsd";
     license = lib.licenses.mit;
-    maintainers = with lib.maintainers; [ d-xo ];
```

Comment:
shouldn't these be left as empty lists?

Reply:
I think it was this thread https://github.com/NixOS/nixpkgs/issues/331085 but there was no proper conclusion.

I feel like leaving it as an empty list can help people realise when doing package bumps or updates that the maintainer list is empty and they can choose to become one.

Whereas removing the entire line will make it hard to notice that it is an unmaintained package like for eg. for many packages the `platforms` attribute is missing which is not a great place to be. (whenever I want to submit a nixpkgs-review result for single package update prs)